src/main/java/seedu/todo/controllers/concerns/Renderer.java

```diff
@@ -14,6 +14,22 @@
 public class Renderer {
     
     /**
+     * Renders an error message in both the console and the input field, leave null or empty string if not needed.
+     * 
+     * @param consoleMessage       Message to be rendered in the console
+     * @param consoleInputValue    Value to display in the input field
+     */
+    public static void showConsoleError(String consoleMessage, String consoleInputValue) {
```

Comment:
Can we abstract a method

renderDisambiguation(replacedCommand, detailedMessage)

So that a call to `renderDisambiguation("add "yoga" at "sunset"", "Your time is invalid!")` will show something like:

Your last command wasn't clear, please fix. <-- Default disambiguation message.
Your time is invalid! <-- Detailed message passed from controller
(consoleInput) add "yoga" at "sunset" <-- replacedCommand passed from controller